components/RuiExpandCollapse/src/RuiExpandCollapse.ts

```diff
@@ -164,11 +164,6 @@ export class RuiExpandCollapse extends LitElement {
           this._collapseableEl.hidden = true;
           this._collapseableEl.classList.add('hide-content');
         }
-        
-        const expandCollapse: HTMLElement | null = this.shadowRoot.querySelector('.expand-collapse');
```

Comment:
removed since this can be set in css rather than inline -> although could be something I missing with LitElements.

Reply:
@terencejeong my thinking behind setting the opacity after the initialising was complete was to stop the component showing before it was 'ready'. If I recall, without this you will see it pop open or closed when the page loads in some cases. If this is not occurring though possibly I'm mis-remembering, but I definitely wrote it intentionally initially

Reply:
Ok thanks for letting me know, I was sure you had it in there for a reason. I'll update the PR :)